Warp website v2 (#312)
broadinstitute/warp: website/src/pages/privacy.md

On this site, we place cookies on your browser to make our system faster.

> The only plugins this site relies on to track the traffic are [@docusaurus/plugin-google-analytics](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-analytics) and [docusaurus-plugin-hotjar](https://github.com/ArtFlag/docusaurus-plugin-hotjar), we only use the traffic data provided by [Google Analystics](https://analytics.google.com/) and [Hotjar](https://www.hotjar.com/) to collect feedback from you and make this site better and faster.

Reviewer: ```suggestion
The only plugins this site relies on to track the traffic are [@docusaurus/plugin-google-analytics](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-analytics) and [docusaurus-plugin-hotjar](https://github.com/ArtFlag/docusaurus-plugin-hotjar). We only use the traffic data provided by [Google Analystics](https://analytics.google.com/) and [Hotjar](https://www.hotjar.com/) to collect feedback from you and make this site better and faster.
```

- Authorized staff on the WARP development team.

- Google Analytics, which WARP docsite relies on to [analyze service usage](https://www.google.com/analytics/terms/us.html). To learn more about how to opt out of Google’s use of cookies as part of its Analytics services, you may visit Google’s Ads Settings. You will be able to prevent your browsing data from being used by Google Analytics by installing Google’s opt-out browser add-on from the Google Analytics Opt-out Page.

> - Hotjar, which WARP docsute uses to collect your feedback. You may visit [Hotjar compliance page](https://help.hotjar.com/hc/en-us/categories/360003405813) to learn more about how they operate.

Reviewer: ```suggestion
Hotjar, which the WARP doc site uses to collect your feedback. You may
```
